submission/submission-core/src/main/kotlin/ac/uk/ebi/biostd/submission/domain/submission/SubmissionProcessor.kt

```diff
@@ -58,7 +59,7 @@ class SubmissionProcessor(
             creationTime = creationTime,
             tags = submission.tags.map { ExtTag(it.first, it.second) },
             collections = tags.map { ExtCollection(it) },
-            section = toExtSectionMapper.convert(accNoString, rqt.version, submission.section, rqt.sources),
+            section = rootSection,
```

Comment:
why this change?

Reply:
To avoid generating a DOI until the submission has been properly mapped i.e. all the files exist